psi4/src/core.cc

```diff
@@ -156,10 +156,10 @@ void py_psi_plugin_close_all();
 
 extern std::map<std::string, plugin_info> plugins;
 
-namespace opt {
-psi::PsiReturnType optking(psi::Options&);
-void opt_clean();
-}  // namespace opt
+// namespace opt {
+// psi::PsiReturnType optking(psi::Options&);
+// void opt_clean();
+// }  // namespace opt
```

Comment:
you can also delete legacy_molecule/wavefunction/gradient . first and last are cpp-optking-only. they're a little spread out -- defined in `libpsi4util/process.h`. not strictly necessary, so up to you.

Reply:
Now that optking isn't using `PsiReturnType.Failure` and `EndLoop` should those be deprecated and/or removed? A quick search indicates success is used by fnocc I didn't see balk used anywhere. I'm happy to perform as much or as little code cleanup with this PR as needed.

Reply:
I think kill off `Balk` and `EndLoop`. `Success` and `Failure` are reasonable for general use.